src/main/java/com/yandex/money/api/model/showcase/components/Component.java

```diff
@@ -8,6 +10,13 @@
  */
 public abstract class Component {
 
+    @Override
+    public final String toString() {
+        return getToStringBuilder().toString();
+    }
+
+    protected abstract ToStringBuilder getToStringBuilder();
```

Comment:
Do you read my comment above?

Pros:
1. No need to create `toStringBuilder` method. We we only override this one. 

Cons:
1. `toStringBuilder` will be `public` (thanks, Java). API should be clean and concise.
2. There's no way to override `toString` in interface. So we should recreate it every time as before.
3. Some `Stringable/Prettified/etc` interface will be present, and top level classes should implement it.